notebooks/tutorials/netcdf-to-cog-cmip6.ipynb

```diff
@@ -0,0 +1,1349 @@
+{
```

Comment:
<p>What is this cell showing? Can you add text or remove?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NASA-IMPACT/veda-docs/pull/71/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='27'/>

Reply:
Good call - added annotation.